modules/references/tests/client/ReferencesNew.client.component.tests.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+import ReferencesNew from '../../client/components/ReferencesNew.component';
+import { Self, Loading, Duplicate, Submitted } from '../../client/components/Info';
```

Comment:
These component names are pretty compact and don't necessarily describe well what they're doing if just looking at them. :-) Could we rename them for more clarity?